action.yml

```
@@ -27,6 +27,13 @@ inputs:
     description: 'Python Version specifier (PEP440) - e.g. "21.5b1"'
     required: false
     default: ""
+outputs:
+  is_formatted:
+    description: "Whether the files were formatted using the black formatter."
+    value: steps
+  changed_files:
+    description: "no. of files black changed"
```

```suggestion
    description: "Number of files black changed."
```
Lets save the ambiguity for non native English speakers.

---

Ill amend the patch with local changes